src/System.Data.SqlClient/src/Microsoft/SqlServer/Server/SmiMetaData.cs

```diff
@@ -139,7 +139,7 @@ internal static SmiMetaData DefaultNVarChar
         }
 
 
-        // SMI V100 (aka V3) constructor.  Superceded in V200.
```

Comment:
On a similar note, the use of periods with abbreviations is mostly a matter of style, and I personally prefer to leave them out, but with the older Latin-based abbreviations using them is more common that not using them so I've been changing to that when I've come across "ie", "eg", "etc", etc.